### ISSUE TITLE
Is there a function similar to torch.abs() ?

### ISSUE THREAD

**Issue by Pomato1999** [NVIDIA/MinkowskiEngine#521]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by TumVink** [comment]
same here.
@Pomato1999 Did you find such functions? or you wrote the function yourself? And could you share the codes if it is convenient for you.

**Comment by Pomato1999** [comment]
yep. You can use torch. abs () to separate the parts that need to be absolute valued, and then reconstruct the sparse vector.
good luck
